packages/hash/api/src/model/ontology/entity-type.model.ts

```diff
@@ -0,0 +1,130 @@
+import { EntityType, GraphApi } from "@hashintel/hash-graph-client";
+
+import { EntityTypeModel, PropertyTypeModel, LinkTypeModel } from "../index";
+
+type EntityTypeArgs = {
```

Comment:
(Low importance) I missed this on the other PRs but perhaps we want to call this EntityTypeModelArgs or something

Reply:
Yeah, I was also thinking we should make it more explicit that this is referring to the class constructor's arguments, and not the static creation method's arguments (for example `EntityType.create(...)`).

@thehabbos007 @Alfred-Mountfield what would you think about:
```suggestion
type EntityTypeModelConstructorArgs = {
```

Reply:
I think it's fine with the verbose name as it's only used for the actual constructor and not in the rest of the code

Reply:
Have addressed in https://github.com/hashintel/hash/pull/903/commits/f7a8d3d650d89c4a3a8e62c21685e71d7e371be0

Reply:
Cheers